### PR TITLE
some rewording

### PR DESCRIPTION
**Pull request by lf-**

### CODE REVIEW

File: _posts/2024-02-09-flipper-zero.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ It will likely be proposed to use the Radiocommunications Act to regulate the ch
 
 It is interesting and worth mentioning that this ban lets automakers get away with having bad and cheap security, while banning tools that would allow individuals to repair and modify their vehicles outside of the manufacturer monopoly, something the manufacturers have been fighting for decades. 
 
-### (This might be convincing to Vancouver area MPs?) This Ban will impact Vancouver's role as a technology hub.
+### (For Vancouver MPs) This Ban will impact Vancouver's role as a technology hub.
```
Comment (omentic) [inline _posts/2024-02-09-flipper-zero.md:66]: lowercase ban?

File: _posts/2024-02-09-flipper-zero.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ It is also a key principle in cybersecurity that security by obsurity is not sec
 
 The Flipper Zero is made out of generic chips costing less than $10, which are exactly the same as the ones used in consumer products with wireless functionality, such as smart light bulbs, garage door openers, and possibly even some cars. Criminals can make purpose-built devices that do the same thing for the purposes of stealing insecure cars or committing other crimes, using these same chips in a new design.
 
-It is very easy to build electronics these days, which is fantastic for innovation. But, this also means that if something is insecure and can be exploited with simple hardware, such hardware is cheap to make. We estimate that a custom device with equivalent wireless functionality to the Flipper could be built as a one-off prototype for $150, and less in higher quantities.
+It is very easy to build electronics these days, which is fantastic for innovation. But, this also means that if something is insecure and can be exploited with simple hardware, such hardware is cheap to make. We estimate that a custom device with equivalent wireless functionality to the Flipper could be built as a one-off prototype for less than $150, and less in higher quantities.
```
Comment (omentic) [inline _posts/2024-02-09-flipper-zero.md:52]: perhaps compare to the flipper itself being $200 (more expensive)

Comment (omentic) [reply _posts/2024-02-09-flipper-zero.md:52]: or don't. i don't know if it adds anything. i did find that kinda funny though

Comment (lf-) [reply _posts/2024-02-09-flipper-zero.md]: lol this is true. i did somewhat pull the numbers out of my ass, but the wireless chip is $10, fancy $10 micro, $40 of whatever else candles or something please help me budget this my BOM is dying, use a PCB antenna, assume PCBs are $60 and we are still way under the estimate.

**Comment by omentic** [comment]
covered in #131

**Pull request closed without merging**